src/sentry/api/event_search.py

```diff
@@ -2021,8 +2031,9 @@ def result_type_fn(function_arguments, parameter_values):
             private=True,
         ),
         Function(
-            "measurements_histogram",
+            "histogram2",
```

Comment:
Is the plan to eventually replace the existing `histogram` with this function? Should we rename the old function to `histogram_deprecated` (or similar) instead?

Reply:
Yes, the intent is to eventually replace the existing `histogram` function. Would you prefer I deprecated the old function as part of this PR?

Reply:
Yeah, that way we can rename `measurements_histogram` to just `histogram` which makes it clearer that its the  right histogram function to use.